webpack.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  entry: "./browser/index.js",
+  output: {
+    path: __dirname + "/build",
+    filename: "bundle.js",
+    module: {
+      loaders: [
+        {
+          test: /\.(js|jsx)$/,
+          exclude: /(node_modules|bower_components)/,
+          loader: "babel-loader",
+          query: {
+            presets: ['es-2015']
```

Comment:
I think you can omit the presets if we have a .babelrc file?  Also, I think we need some react presets...